### PR TITLE
Ajout du script bye-benfoul.php

### PR DESCRIPTION
**Pull request by benFoulon**
Message d'au revoir

### CODE REVIEW

File: bye-benfoulon.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?php
+
+echo 'bye BenFoulon';
```
Comment (francoismk) [inline bye-benfoulon.php:3]: :heart: :hear_no_evil:

Comment (Baubaux) [reply bye-benfoulon.php:3]: :1st_place_medal:

**Comment by francoismk** [comment]
:+1:

File: bye-benfoulon.php (hunk 1 above)
Comment (benFoulon) [reply bye-benfoulon.php:3]: :wink:

**Comment by Baubaux** [comment]
:+1:

**Pull request merged**